src/module/apps/dialogs/ShadowrunActorDialogs.ts

```diff
@@ -140,55 +133,12 @@ export class ShadowrunActorDialogs {
             },
         };
 
-        const onAfterClose = (html: JQuery) => {
-            const soak: DamageData = options?.damage
-                    ? options.damage
-                : DefaultValues.damageData({type: {base: '', value: ''}});
-            // handle ap changes
-            const ap = Helpers.parseInputToNumber($(html).find('[name=ap]').val());
-
-
-            const armor = actor.getArmor();
-            if (armor) {
-                // handle element changes
-                const element = Helpers.parseInputToString($(html).find('[name=element]').val());
-                if (element) {
-                    soak.element.value = element as DamageElement;
-                }
-                const bonusArmor = armor[element] ?? 0;
-                if (bonusArmor) {
-                    parts.addUniquePart(CONFIG.SR5.elementTypes[element], bonusArmor);
-                }
-
-                if (ap) {
-                    let armorVal = armor.value + bonusArmor;
-
-                    // don't take more AP than armor
-                    parts.addUniquePart('SR5.AP', Math.max(ap, -armorVal));
-                }
-            }
-
-            // handle incoming damage changes
+        const onAfterClose = (html: JQuery) => {           
             const incomingDamage = Helpers.parseInputToNumber($(html).find('[name=incomingDamage]').val());
-            if (incomingDamage) {
-                const totalDamage = Helpers.calcTotal(soak);
-                if (totalDamage !== incomingDamage) {
-                    const diff = incomingDamage - totalDamage;
-                    // add part and calc total again
-                    soak.mod = PartsList.AddUniquePart(soak.mod, 'SR5.UserInput', diff);
-                    soak.value = Helpers.calcTotal(soak);
-                }
-
-                const totalAp = Helpers.calcTotal(soak.ap);
-                if (totalAp !== ap) {
-                    const diff = ap - totalAp;
-                    // add part and calc total
-                    soak.ap.mod = PartsList.AddUniquePart(soak.ap.mod, 'SR5.UserInput', diff);
-                    soak.ap.value = Helpers.calcTotal(soak.ap);
-                }
-            }
+            const ap = Helpers.parseInputToNumber($(html).find('[name=ap]').val());
+            const element = Helpers.parseInputToString($(html).find('[name=element]').val());
 
-            return {soak, parts};
+            return {incomingDamage: incomingDamage, ap: ap, element: element};
```

Comment:
Ultra Nitpick: Kurzform Syntax {incomingDamage, ap, element}.

Reply:
Will add it this in the next PR